Explorer/Assets/DCL/SkyBox/Plugin/TimeOfDaySystem.cs

```diff
@@ -92,9 +92,9 @@ private void CompleteLightJob()
 
             light.transform.localRotation = output.LightRotation;
             //light.intensity = output.LightIntensity;
-            Color ambientSkyColor = new Color((float)125 / 255, (float)136 / 255, (float)159 / 255);
-            Color ambientEquatorColor = new Color((float)170 / 255, (float)197 / 255, (float)178 / 255);
-            Color ambientGroundColor = new Color((float)12 / 255, (float)9 / 255, (float)3 / 255);
+            //Color ambientSkyColor = new Color((float)125 / 255, (float)136 / 255, (float)159 / 255);
+            //Color ambientEquatorColor = new Color((float)170 / 255, (float)197 / 255, (float)178 / 255);
+            //Color ambientGroundColor = new Color((float)12 / 255, (float)9 / 255, (float)3 / 255);
```

Comment:
I commented them because they were not being used, but following these ones there is MORE commented code that I do not know why its commented (had no comment explaining it xD).
Just commented these to avoid doing unnecessary calculations that were not being used at all...